src/index.js

```diff
@@ -41,12 +41,18 @@ ReactGA.initialize('UA-532159-1');
 
 const history = createBrowserHistory();
 
-// Get the current location
-history.listen((location) => {
+const trackNavigation = (location) => {
   const locationPage = location.hash && location.hash[0] === '#' && location.hash.slice(1);
   ReactGA.set({ page: locationPage });
   ReactGA.pageview(locationPage);
```

Comment:
I understand that removing _locationPage_ parameter from here is out of the scope of this PR.

Reply:
I don't get your point here. Why remove this `locationPage`?

Reply:
I'm obligued to add location page in pageview, by this library we are using now. I've tried with no luck. `ReactGA.pageview()`;

Reply:
👍